library/src/main/java/com/anjlab/android/iab/v3/SkuDetails.java

```diff
@@ -36,7 +37,17 @@
 
 	public final String currency;
 
-	public final Double priceValue;
+	public final double priceValue;
+
+	public final Period subscriptionPeriod;
+
+    public final Period subscriptionFreeTrialPeriod;
```

Comment:
Sure. My first idea was use default android sdk's class [documentation](https://developer.android.com/reference/javax/xml/datatype/DatatypeFactory.html)
but unfortunately android haven't default implementation and that means that android can't parse ISO 8601 date periods. So there was 2 ways: implement custom parser and use 3rd party library. Joda Time is a good alternative I think (honestly I don't found any other libraries). If you know better library which work with ISO 8601 you can suggest it.